src/MenuState.cpp

```diff
@@ -26,7 +26,7 @@ MenuState::MenuState(bool main_menu){
 	background = Sprite("menu/background.jpg");
 	title = Sprite("menu/title.png", 5, FRAME_TIME);
 	planet = Sprite("menu/planet.png", 8, FRAME_TIME);
-	green_ship = Sprite("menu/green_ship.png", 8, FRAME_TIME, 2);
+	green_ship = Sprite("menu/green_ship.png", 8, FRAME_TIME);
```

Comment:
It would be nice to let the green ship start at a different frame than the red ship, so it could give a sense of asynchrony between them.